README.md

```diff
@@ -118,7 +118,7 @@ The [scripts/run-artifactory.sh](scripts/run-artifactory.sh) starts two Artifact
 
 Set the env var to the second Artifactory instance URL. This is the URL that will be accessible from `artifactory-1` container (not the URL from the Docker host):
 ```sh
-$ export ARTIFACTORY_URL_2=http://localhost:9081
+$ export ARTIFACTORY_URL_2=http://artifactory-2:8082
```

Comment:
is this correct? 8082 will be already taken by the first instance

Reply:
Yes, this is the URL used by the first docker instance to access the second one. They are in the same Docker network so can be accessed via the service name and port.